sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisSuite.scala

```diff
@@ -1183,7 +1192,7 @@ class AnalysisSuite extends AnalysisTest with Matchers {
         |    GROUP BY a, b
         |)
       """.stripMargin),
-      Seq("grouping_id() can only be used with GroupingSets/Cube/Rollup"),
+      Seq("grouping()/grouping_id() can only be used with GroupingSets/Cube/Rollup"),
```

Comment:
Use the function which checks the error class and its parameters.

Reply:
Got it.